### DISCUSSION TITLE
Differentiate files with the same name in bufferline

### DISCUSSION THREAD

**Discussion by norman784** [helix-editor/helix#10191] [Ideas]
I think this will make a QoL improvement, now I updated and tried the bufferline and this is the first issue found. For example in a cargo workspace you have multiple `lib.rs`, `main.rs` or also a normal project `mod.rs` files. Maybe we can draw some inspiration from VScode in this regard, I'm not sure how the bufferline.nvim solves it.

**Comment by mageowl** [comment]
I have a pull request that fixes this issue: #13344
It still needs review, but you can use it for yourself for now.
